addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -0,0 +1,541 @@
+import { start } from "@mail/../tests/mail_test_helpers";
+import { contains, mountView, onRpc, makeMockServer } from "@web/../tests/web_test_helpers";
+import { getOrigin } from "@web/core/utils/urls";
+import { describe, expect, test } from "@odoo/hoot";
+import { queryAll, queryAllAttributes, queryOne, queryFirst } from "@odoo/hoot-dom";
+import { defineHrModels } from "@hr/../tests/hr_test_helpers";
+
+describe.current.tags("desktop");
+defineHrModels();
+
+test("many2one_avatar_employee widget in list view", async () => {
+    const { env } = await makeMockServer();
+    const [partnerId_1, partnerId_2] = env["res.partner"].create([
+        { name: "Mario" },
+        { name: "Luigi" },
+    ]);
+    const [userId_1, userId_2] = env["res.users"].create([
+        { partner_id: partnerId_1 },
+        { partner_id: partnerId_2 },
+    ]);
+    const [employeeId_1, employeeId_2] = env["hr.employee.public"].create([
+        {
+            name: "Mario",
+            user_id: userId_1,
+            user_partner_id: partnerId_1,
+            work_email: "Mario@partner.com",
+        },
+        {
+            name: "Luigi",
+            user_id: userId_2,
+            user_partner_id: partnerId_2,
+        },
+    ]);
+    env["m2x.avatar.employee"].create([
+        {
+            employee_id: employeeId_1,
+            employee_ids: [employeeId_1, employeeId_2],
+        },
+        { employee_id: employeeId_2 },
+        { employee_id: employeeId_1 },
+    ]);
+    await start();
+    onRpc("has_group", () => false);
+    await mountView({
+        type: "list",
+        resModel: "m2x.avatar.employee",
+        arch: `<list><field name="employee_id" widget="many2one_avatar_employee"/></list>`,
+    });
+    expect(queryFirst(".o_data_cell div[name='employee_id']").innerText).toBe("Mario");
+    expect(queryAll(".o_data_cell div[name='employee_id']")[1].innerText).toBe("Luigi");
+    expect(queryAll(".o_data_cell div[name='employee_id']")[2].innerText).toBe("Mario");
```

Comment:
```suggestion
    expect(".o_data_cell div[name='employee_id']:eq(0)").toHaveText("Mario");
    expect(".o_data_cell div[name='employee_id']:eq(1)").toHaveText("Luigi");
    expect(".o_data_cell div[name='employee_id']:eq(2)").toHaveText("Mario");
```

Same remark for other equivalent occurrences of `el.textContent|innerText.toBe()` => `expect(selector).toHaveText()`